build.sbt

```diff
@@ -14,18 +14,9 @@ import scala.util.Try
 import scala.xml.Elem
 import scala.xml.transform.{RewriteRule, RuleTransformer}
 
-// Warning: Flink doesn't work correctly with 2.12.11
-// Warning: 2.12.13 + crossVersion break sbt-scoverage: https://github.com/scoverage/sbt-scoverage/issues/319
-val scala212 = "2.12.10"
+val scala212 = "2.12.16"
```

Comment:
I think scala bump is still not possible: https://lists.apache.org/thread/hwksnsqyg7n3djymo7m1s7loymxxbc3t

Reply:
still? what's a pity. ok, I've removed scala upgrade from this PR